packages/mui-material/src/Alert/Alert.js

```diff
@@ -202,8 +204,9 @@ const Alert = React.forwardRef(function Alert(inProps, ref) {
             title={closeText}
             color="inherit"
             onClick={onClose}
+            {...componentsProps.closeAction}
```

Comment:
`closeIcon` (`componentsProps.closeIcon`) would be a better name because `action` is null here.

Reply:
I was considering this, but wouldn't that suggest that the props would be applied to the `<CloseIcon />` instead of the `<IconButton />`? Would naming it `componentsProps.closeIconButton` be better?

Reply:
I think we can include two slots:

- For IconButton (`componentsProps.closeIconButton`)
- For CloseIcon (`componentsProps.closeIcon`)

It is not necessary to have a corresponding `components` slot for a `componentsProps` slot.